sw/device/silicon_creator/manuf/base/perso_tlv_data.h

```diff
@@ -178,35 +178,38 @@ rom_error_t perso_tlv_get_cert_obj(const uint8_t *buf, size_t ltv_buf_size,
  * |                   cert                       |
  * +----------------------------------------------+
  *
- *
- * @param name the name of the certificate
- * @param needs_endorsement defines the type of the LTV object the certificate
- *                          is wrapped into
- * @param cert_body the actual certificate
- * @param cert_size size of the certificate in bytes
- * @param[out] perso_blob container for sending data to host.
- *
+ * @param name The name of the certificate.
+ * @param needs_endorsement Defines the type of the LTV object the certificate
+ *                          is wrapped into (TBS or fully formed).
+ * @param cert The binary certificate blob.
+ * @param cert_size Size of the certificate blob in bytes.
+ * @param[out] buf Output buffer to copy the data into.
+ * @param[inout] buf_size Input is size of the output buffer in bytes; output is
+ *                        space of buffer that was consumed by the LTV object.
  * @return status of the operation.
  */
 OT_WARN_UNUSED_RESULT
-status_t perso_tlv_prepare_cert_for_shipping(const char *name,
```

Comment:
It requires to update the perso_extension as well.

Reply:
yes I am aware. I will be submitting a corresponding PR on that front too.